wp-content/themes/cf-voyager/lessons-cpt.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Template Name: Lessons CPT
+ *
+ */
+
+get_header();
+
+$custom_terms = get_terms('lesson_types');
+
+foreach ($custom_terms as $custom_term) {
+	wp_reset_query();
```

Comment:
You do not need to use `wp_reset_query` if you are using a custom `WP_Query`. `wp_reset_query` is needed when the main query is overwritten (using `query_posts`). See: https://developer.wordpress.org/reference/functions/wp_reset_query/

Instead of using a tax_query, use a meta_query to get all lessons having the specific meta value, as requested in the ticket. Do not check the meta value for all results like on line 43 as this approach will filter the 5 results returned by the query.